boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -263,6 +263,9 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
 
     while (PcBiosMapCount < MAX_BIOS_DESCRIPTORS)
     {
+        /* ACPI 3.0/4.0: Set Extended Attributes to enabled/valid by default, in case entry has no E.A.. */
+        *(ULONG *)&((BIOS_MEMORY_MAP *)BIOSCALLBUFFER)->EA = 0x00000001;
```

Comment:
Just add an AsULONG union member or similar, much like you had with Reserved. Then you don't need to cast this.

Reply:
Also, the 1 shouldn't be a magic number here, set it to 0 and then set the bit via the bitfield.